R/update-news.R

```diff
@@ -19,39 +20,78 @@ collect_news <- function(messages) {
     purrr::discard(~ . == "") %>%
     purrr::map_chr(remove_housekeeping) %>%
     purrr::map(extract_newsworthy_items) %>%
-    unlist()
+    purrr::keep(~ nrow(.) > 0) %>%
+    bind_rows()
 
-  if (length(newsworthy_items) == 0) {
+  if (is.null(newsworthy_items)) {
     if (length(messages) <= 1) {
-      newsworthy_items <- "- Same as previous version."
+      newsworthy_items <- parse_regular_commit("Same as previous version.")
```

Comment:
the `parse_blabla_commits()` functions return tibbles with description, type, breaking, scope.